libraries/docker_network.rb

```diff
@@ -1,3 +1,5 @@
+require 'ipaddress'
```

Comment:
This should be moved into ``DockerCookbook::DockerNetwork``.

Reply:
Could you point me to the style guide for sous-chef cookbooks that recommends this practise? [StackOverflow recommends to put require at the top](https://stackoverflow.com/a/605329), since the namespaces are imported to the toplevel anyway and it's the most common convention.

Reply:
We really don't have a style guide other than what `cookstyle` provides, however I was mostly looking at the style/standard this cookbook uses and trying to stick to that.

Reply:
Okay, consistency with the surrounding code convinces me. Done.